advisories/unreviewed/2022/05/GHSA-x534-j49x-mqvj/GHSA-x534-j49x-mqvj.json

```diff
@@ -1,19 +1,43 @@
 {
   "schema_version": "1.4.0",
   "id": "GHSA-x534-j49x-mqvj",
-  "modified": "2023-03-01T18:30:58Z",
+  "modified": "2023-03-09T05:05:46Z",
```

Comment:
Not sure why this `modified` value changed, and why it uses a date from 2023. I just copied this from #5161 where it was automatically changed like this.